server/@types/express/index.d.ts

```diff
@@ -44,11 +45,10 @@ declare module 'express-session' {
     waitListApplicationJourney: WaitListApplicationJourney
     notAttendedJourney: NotAttendedJourney
     recordAttendanceRequests: RecordAttendanceRequests
+    journeyMetrics: JourneyMetrics
```

Comment:
Instead of overloading journey sessions with generic metrics like journey start time, this holds metrics in a separate object mapped by the journey id (if one exists).